README.md

```diff
@@ -244,6 +244,40 @@ type Output struct {
 }
 ```
 
+### Struct field name folding
+
+With `goverter:fold` tag you can instruct goverter to perform case-insensitive mapping between
+source and target fields. This is useful when converting between two structs auto-generated by
+two different tools, each following different name casing rules for struct fields: goverter
+will match the fields using strings.EqualFold method if fold tag is present.
+
+Use this tag only when it is extremely unlikely for the source or the target to have two fields
```

Comment:
Do you mean "two *different* fields' otherwise it reads like that it shouldn't be used for mapping fields that have different casing?

Reply:
ah, I just removed this piece, I think the use case in second paragraph already covers the details.